contracts/0.8.25/upgrade/V3VoteScript.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import {IBurner} from "contracts/common/interfaces/IBurner.sol";
+
+import {OmnibusBase} from "./OmnibusBase.sol";
+import {IOssifiableProxy} from "./interfaces/IOssifiableProxy.sol";
+import {IAccessControl} from "@openzeppelin/contracts-v5.2/access/IAccessControl.sol";
+import {V3Addresses} from "./V3Addresses.sol";
+
+interface IRepo {
+    function newVersion(uint16[3] calldata _newSemanticVersion, address _contractAddress, bytes calldata _contentURI) external;
+}
+
+interface IKernel {
+    function setApp(bytes32 _namespace, bytes32 _appId, address _app) external;
+    function APP_BASES_NAMESPACE() external view returns (bytes32);
+}
+
+interface IStakingRouter {
+    function REPORT_REWARDS_MINTED_ROLE() external view returns (bytes32);
+}
+
+interface ITemplateV3 {
+    function startUpgrade() external;
+    function finishUpgrade() external;
+}
+
+/// @title V3VoteScript
+/// @notice Script for upgrading Lido protocol components
+contract V3VoteScript is OmnibusBase {
```

Comment:
Need to discuss and decide what we will do with:
- new ET plugs
- initial parameters, caps, and limits